libraries/SITL/SITL.h

```diff
@@ -165,6 +166,9 @@ class SIM {
 #ifdef SFML_JOYSTICK
     static const struct AP_Param::GroupInfo var_sfml_joystick[];
 #endif //SFML_JOYSTICK
+#if AP_STATS_ENABLED
+    AP_Stats node_stats;
```

Comment:
Not used

Reply:
3a172aab4f33045c1e96c7dfc35b3a82876e881f